x-pack/platform/plugins/shared/cases/server/client/alerts/get.ts

```diff
@@ -37,3 +42,41 @@ export const getAlerts = async (
     ...alert._source,
   }));
 };
+
+export const getAlertMetadataFromComments = async (
+  comments: AttachmentAttributes[],
+  clientArgs: CasesClientArgs
+): Promise<CaseMetadata> => {
+  const alertInfo: AlertInfo[] = comments.flatMap((c) => {
```

Comment:
nit: prefer full names: 
```suggestion
  const alertInfo: AlertInfo[] = comments.flatMap((comment) => {
```